File: s.cpp
#include <iostream>

#include <chrono>

double calc (double x) {
    return x*x - x*x + x*4 - x*5 + x + x;
}



int main()

{
    long long n;
    
    double res;
    
    double  time;
    
    int x = 5;

    while(1)
    {
        if (!(std::cin >> n)) 
            break; 
        
        clock_t start = clock();

        for (long long i =0; i<n; ++i)
        {
            res = calc(x);
        }

        clock_t end = clock();
        time = (double)(end - start) / CLOCKS_PER_SEC;

        std::cout <<"Время =  "<< time << "\n";
        std::cout <<"Итог =  "<< res <<std:: endl;
    }
}

File: s.py
import time

def calc(x):
    return x**2 - x**2 + x*4 - x*5 + x + x

x = 10

while 1==1:
    n = input()

    if not n.isdigit():
        break
    
    n = int(n)
    
    start_time = time.time()

    res = 0

    for i in range(n):
        res = calc(x)
        
    end_time = time.time()
    
    duration = end_time - start_time
    
    print(f"Время : {duration} s")
    print(f"Итог : {res}")
